spec/controllers/events_controller_spec.rb

```diff
@@ -1,6 +1,13 @@
 require 'rails_helper'
 
 RSpec.describe EventsController, type: :controller do
+  let(:admin_person) { FactoryGirl.build(:person, :admin) }
+
+  before do
+    allow_any_instance_of(EventsController).to receive(:pundit_user)
+      .and_return(admin_person)
+  end
```

Comment:
I think the only way I could do that would be to write a comment explaining my actions. Gray also didn't know why this was here so that might be a good idea.